content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -185,6 +184,33 @@ parallelism, for a variety of reasons:
 - The Job controller may throttle new Pod creation due to excessive previous pod failures in the same Job.
 - When a Pod is gracefully shut down, it takes time to stop.
 
+#### Completion Mode
```

Comment:
BTW, this would work better as a level 3 heading so that it's part of in-page navigation.
```suggestion
### Completion mode
```

Reply:
Done